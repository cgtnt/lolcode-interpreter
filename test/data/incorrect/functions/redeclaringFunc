HAI

HOW IZ I FOO
  FOUND YR 1
IF U SAY SO

HOW IZ I FOO
  FOUND YR 2
IF U SAY SO

KTHXBYE
