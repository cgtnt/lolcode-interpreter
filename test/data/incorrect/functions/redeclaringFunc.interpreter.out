Line 7: Cannot redefine variable FOO
